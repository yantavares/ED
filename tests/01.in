5
A .
C -
R .-
T ..
! -.
0
. .. . - . .- -.