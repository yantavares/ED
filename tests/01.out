ATACAR!
* A C T R !